src/Illuminate/Contracts/Database/Eloquent/Factory.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Illuminate\Contracts\Database\Eloquent;
+
+use ArrayAccess;
+use Faker\Generator as Faker;
+
+interface Factory extends ArrayAccess
+{
+    /**
+     * Create a new factory container.
+     *
+     * @param  \Faker\Generator  $faker
+     * @param  string|null  $pathToFactories
+     * @return Factory
+     */
+    public static function construct(Faker $faker, $pathToFactories = null);
```

Comment:
I'm not sure this should be in a contract?

Reply:
I removed it.